sdk/core/src/index.ts

```diff
@@ -213,6 +216,8 @@ var ReactMoE = {
   /**
    * Update user's unique id which was previously set by setUserUniqueID()
    * @param alias updated unique id.
+   * 
+   * @deprecated This function is deprecated in favour of identifyUser(). This function will be removed in 13.0.0"
    */
   setAlias: function (alias: string) {
```

Comment:
Let's add a log for deprecated method usage

Reply:
added